exporters/pelorus/deserialization/__init__.py

```diff
@@ -311,14 +387,12 @@ def _deserialize_field(self, src: Mapping[str, Any], field: attrs.Attribute):
 
     def _deserialize_attrs_class(self, src: Mapping[str, Any], to: type[T]) -> T:
         "Initialize an attrs class field-by-field."
-        cls = _is_attrs_class(to)
-        assert cls is not None, f"class was not an attrs class: {to.__name__}"
+        assert _is_attrs_class(to), f"class was not an attrs class: {to.__name__}"
```

Comment:
should we change the assert here? since the compiler can ignore it in an optimized build

Reply:
I don't think we use optimizations. And this is more for peace of mind-- if I've written the code right, this case should never happen anyway.